packages/wonder-blocks-icon-button/src/components/icon-button-core.tsx

```diff
@@ -353,29 +362,11 @@ const _generateStyles = (
                     backgroundColor: "transparent",
                 },
             },
-            // Provide basic, default focus styles on older browsers (e.g.
-            // Safari 14)
-            ":focus": {
-                boxShadow: `0 0 0 ${theme.border.width.default}px ${defaultStrokeColor}`,
-                borderRadius: theme.border.radius.default,
-            },
-            // Remove default focus styles for mouse users ONLY if
-            // :focus-visible is supported on this platform.
-            ":focus:not(:focus-visible)": {
-                boxShadow: "none",
-            },
-            // Provide focus styles for keyboard users on modern browsers.
-            ":focus-visible": {
-                // Reset default focus styles
-                boxShadow: "none",
-                // Apply modern focus styles
-                outlineWidth: theme.border.width.default,
-                outlineColor: defaultStrokeColor,
-                outlineOffset: 1,
-                outlineStyle: "solid",
-                borderRadius: theme.border.radius.default,
-                ...kindOverrides[":focus-visible"],
-            },
+            // :focus -> Provide basic, default focus styles to support
+            // programmatic focus (e.g. element.focus()).
+            ":focus": focusStyles,
```

Comment:
Normally it could be an issue for `:focus` to persist on click as sighted users sometimes complain about it...but in this case clicking the button closes the modal, and the entire thing goes away. I agree that it might make sense to limit this change to the modal close buttons, and not `icon-button-core`. That would require quite a bit more testing!

Reply:
ok, I've applied the changes, let me know what you think.